layers/+email/mu4e/packages.el

```diff
@@ -46,6 +47,10 @@
         (add-hook 'mu4e-compose-pre-hook 'mu4e/set-account)
         (add-hook 'message-sent-hook 'mu4e/mail-account-reset)))))
 
+(defun mu4e/init-mu4e-alert ()
+  (use-package mu4e-alert
+    :commands (mu4e-alert-enable-notifications)))
+
```

Comment:
`:commands` is not needed as the function already has autoload defined. 
You may also add layer variables to turn on/off notifications/mode-line by adding following snippet in `:init`
```elisp
(with-eval-after-load 'mu4e
  (when mu4e-enable-notifications
    (mu4e-alert-enable-notifications))
  (when mu4e-enable-mode-line
    (mu4e-alert-enable-mode-line-display)))
```

And define layer variables in `config.el`
```elisp 
(defvar mu4e-enable-notifications nil
  "If non-nil, enable desktop notifications for unread emails.")
(defvar mu4e-enable-mode-line nil
  "If non-nil, enable display of unread emails in mode-line.")
```